src/omotes_simulator_core/adapter/transforms/mappers.py

```diff
@@ -193,7 +193,7 @@ def _get_junction(self) -> dict[str, list[tuple[str, str]]]:
             # Esdl Junctions need to be skipped for now, are added later.
```

Comment:
Why are the junctions skipped for now? I thought we had a replacement object to change the joints?

Reply:
indeed, but that is somewhere els ein the code

Reply:
Ok could you make an issue to also convert the joints?